src/main/kotlin/wu/seal/jsontokotlin/utils/Extensions.kt

```diff
@@ -184,6 +184,37 @@ fun theSamePrimitiveType(first: JsonPrimitive, second: JsonPrimitive): Boolean {
     return sameBoolean || sameNumber || sameString
 }
 
+
+/**
+ * Return the kotlin type of a JsonArray with all elements are numbers
+ */
+fun getKotlinNumberClass(jsonArray: JsonArray) : KotlinClass {
+    var ans : KotlinClass =  KotlinClass.INT
+    jsonArray.forEach {
+        if (it.isJsonPrimitive.not() || it.asJsonPrimitive.isNumber.not()) {
```

Comment:
Unused code?

Reply:
@wuseal  it‘s used in GenericListClassGeneratorByJSONArray.kt  and ListClassGeneratorByJSONArray.kt

Reply:
![image](https://user-images.githubusercontent.com/9211902/134022641-74f698a1-0af4-4902-8d7a-7821cf1f96c9.png)
I mean here

Reply:
just added